synfig-studio/src/gui/app.cpp

```diff
@@ -285,6 +285,7 @@ std::set< String > studio::App::brushes_path;
 String studio::App::image_editor_path;
 
 String studio::App::sequence_separator(".");
+int studio::App::number_of_cores(g_get_num_processors());
```

Comment:
cosmetic: it isn't related to surrounding variables. Move it to below ` String studio::App::workarea_renderer; ` separated by a blank line :)